specification/operationalinsights/resource-manager/readme.md

```diff
@@ -150,13 +150,13 @@ input-file:
 - Microsoft.OperationalInsights/stable/2020-08-01/OperationStatuses.json
 - Microsoft.OperationalInsights/stable/2020-08-01/SharedKeys.json
 - Microsoft.OperationalInsights/stable/2020-08-01/Usages.json
-- Microsoft.OperationalInsights/stable/2020-08-01/StorageInsightConfigs.json
 - Microsoft.OperationalInsights/stable/2020-08-01/SavedSearches.json
 - Microsoft.OperationalInsights/stable/2020-08-01/AvailableServiceTiers.json
 - Microsoft.OperationalInsights/stable/2020-08-01/Gateways.json
 - Microsoft.OperationalInsights/stable/2020-08-01/Schema.json
 - Microsoft.OperationalInsights/stable/2020-08-01/SharedKeys.json
 - Microsoft.OperationalInsights/stable/2020-08-01/WorkspacePurge.json
+- Microsoft.OperationalInsights/stable/2020-10-01/StorageInsightConfigs.json
```

Comment:
> Microsoft.OperationalInsights/stable/2020-10-01/StorageInsightConfigs.json [](start = 1, length = 75)

This replacement is possibly will break the customers using 2020-10 SDK version since now the StorageAccount model is changed in the new StorageInsightConfigs definition.